plexupdate.sh

```diff
@@ -54,7 +54,7 @@ AUTOUPDATE=no
 AUTOSTART=no
 ARCH=$(uname -m)
 # patch for Raspberry Pi reporting as armv7l, whereas Plex only offers armv7hf_neon
```

Comment:
Sorry for the dumbest nitpick ever, but would you mind also updating the comment line here with the new name and I’ll get this merged right away? (I just know it’ll drive me nuts at some point down the line when I see that they don’t match.)

Reply:
... done.
:-)